pvlib/tests/test_pvsystem.py

```diff
@@ -1426,6 +1429,13 @@ def test_singlediode_series_ivcurve(cec_module_params):
         assert_allclose(v, expected[k], atol=1e-6)
 
 
+@pytest.mark.parametrize('method', ['lambertw', 'brentq', 'newton'])
```

Comment:
```suggestion
@fail_on_pvlib_version('0.11')
@pytest.mark.parametrize('method', ['lambertw', 'brentq', 'newton'])
```

To remind us to remove this when the deprecation period is over